docs/openapi.json

```diff
@@ -2550,15 +2550,36 @@
             "nullable": false
           },
           "keys": {
-            "type": "integer",
+            "oneOf": [
+              {
+                "type": "string"
+              },
+              {
+                "type": "number"
+              }
+            ],
```

Comment:
apparently we have two experts on this topic now!

Reply:
cut-n-paste!